EasyRestProjectNetTeam2/EasyRestPages/MenuPage.cs

```diff
@@ -6,14 +6,13 @@ namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class MenuPage : BasePage
     {
-        public LeftBarComponent LeftBarComponent { get; set; }
-       
+        public DatePickerComponent DatePickerComponent { get; set; }
```

Comment:
don't use setter for components because someone might re-initialize the component on the test level